app/budget/directives/courseCosts/courseCosts.html

```diff
@@ -34,23 +34,87 @@
 					</td>
 					<!-- Enrollment -->
 					<td class="course-costs__table-cell">
-						{{ sectionGroup.totalSeats }}
+						<div class="course-costs__edit-container">
+							<div class="course-costs__number-input">
+								<ipa-input
+									mode="'number'"
+									value="sectionGroup.overrideTotalSeats"
+									on-update="overrideSectionGroup(sectionGroup, 'seats')">
+								</ipa-input>
+							</div>
+							<div class="glyphicon glyphicon-repeat course-costs__revert-ui"
+								uib-tooltip="Revert to schedule total seats of {{ !!(sectionGroup.totalSeats) ? sectionGroup.totalSeats : 'none' }}"
+								confirm-button="revertOverride(sectionGroup, 'seats')"
+								message="Revert to schedule total seats of {{ !!(sectionGroup.totalSeats) ? sectionGroup.totalSeats : 'none' }}?"
+								yes="Revert"
+								no="Cancel"
+								ng-show="!!(sectionGroup.sectionGroupCost) && sectionGroup.sectionGroupCost.enrollment !== null && (sectionGroup.totalSeats != sectionGroup.sectionGroupCost.enrollment)">
+							</div>
+						</div>
 					</td>
 					<!-- Section Count -->
 					<td class="course-costs__table-cell">
-						{{ sectionGroup.sectionCount }}
+						<div class="course-costs__edit-container">
+							<div class="course-costs__number-input">
+								<ipa-input
+									mode="'number'"
+									value="sectionGroup.overrideSectionCount"
+									on-update="overrideSectionGroup(sectionGroup, 'sectionCount')">
+								</ipa-input>
+							</div>
+							<div class="glyphicon glyphicon-repeat course-costs__revert-ui"
+								uib-tooltip="Revert to schedule section count of {{ !!(sectionGroup.sectionCount) ? sectionGroup.sectionCount : 'none' }}"
+								confirm-button="revertOverride(sectionGroup, 'sectionCount')"
+								message="Revert to schedule section count of {{ !!(sectionGroup.sectionCount) ? sectionGroup.sectionCount : 'none' }}?"
+								yes="Revert"
+								no="Cancel"
+								ng-show="!!(sectionGroup.sectionGroupCost) && sectionGroup.sectionGroupCost.sectionCount !== null && sectionGroup.sectionCount != sectionGroup.sectionGroupCost.sectionCount">
+							</div>
+						</div>
 					</td>
 					<!-- TA Count -->
 					<td class="course-costs__table-cell">
-						{{ sectionGroup.teachingAssistantAppointments || "0" }}
+						<div class="course-costs__edit-container">
+							<div class="course-costs__number-input">
+								<ipa-input
+									mode="'number'"
+									value="sectionGroup.overrideTeachingAssistantAppointments"
+									on-update="overrideSectionGroup(sectionGroup, 'teachingAssistantAppointments')">
+								</ipa-input>
+							</div>
+							<div class="glyphicon glyphicon-repeat course-costs__revert-ui"
+								uib-tooltip="Revert to schedule TA count of {{ !!(sectionGroup.teachingAssistantAppointments) ? sectionGroup.teachingAssistantAppointments : 'none' }}"
```

Comment:
These should line up with 'class' but you are using indents, so they won't depending on the text editor.